tests/cupy_tests/core_tests/test_userkernel.py

```diff
@@ -82,3 +83,21 @@ def test_scalar(self, xp, dtype):
         else:
             kernel = cupy.ElementwiseKernel('T x, T y', 'T z', 'z = x + y')
             return kernel(x, self.value)
+
+
+class TestUserkernelManualBlockSize(unittest.TestCase):
+
+    def test_invalid_block_size(self):
+        x = testing.shaped_arange((2, 3, 4), cupy, cupy.float32)
+        kernel = cupy.ElementwiseKernel('T x, T y', 'T z', 'z = x + y')
+        with pytest.raises(ValueError):
+            kernel(x, 1, block_size=0)
+
+    @testing.numpy_cupy_array_equal()
+    def test_block_size(self, xp):
+        x = testing.shaped_arange((2, 3, 4), xp, xp.float32)
+        if xp is numpy:
+            y = numpy.array(1).astype(xp.float32)
+            return x + y
+        kernel = cupy.ElementwiseKernel('T x, T y', 'T z', 'z = x + y')
+        return kernel(x, 1, block_size=1)
```

Comment:
As the objective of this test is not for comparing with numpy, instead of using `numpy_cupy_array_equal`, how about using `testing.assert_array_equal`?
```py
y = kernel(x, 1, block_size=1)
testing.assert_array_equal(y, x + 1)
```

Reply:
Done, thanks!